common/dds_specific/srcC/DynamicDataHelpers.c

```diff
@@ -0,0 +1,491 @@
+/******************************************************************************/
+/* (c) 2020 Copyright, Real-Time Innovations, Inc. (RTI) All rights reserved. */
+/*                                                                            */
+/* RTI grants Licensee a license to use, modify, compile, and create          */
+/* derivative works of the software solely for use with RTI Connext DDS.      */
+/* Licensee may redistribute copies of the software provided that all such    */
+/* copies are subject to this license.                                        */
+/* The software is provided "as is", with no warranty of any type, including  */
+/* any warranty for fitness for any purpose. RTI is under no obligation to    */
+/* maintain or support the software.  RTI shall not be liable for any         */
+/* incidental or consequential damages arising out of the use or inability to */
+/* use the software.                                                          */
+/*                                                                            */
+/******************************************************************************/
+
+#include <ndds/ndds_c.h>
+
+#include "DynamicDataHelpers.h"
+#include "TransformationLog.h"
+
+DDS_TypeCode * RTI_COMMON_TypeCode_get_member_type(
```

Comment:
I used `RTI_COMMON_` to be consistent with what we have in the `dds_specific/srcCxx` that has namespaces `rti::common`. We decided not to add anything related to the GW because, in the end, these are general functions, they are not tied to the RTI Gateway.
How about `RTI_Common` ?

Reply:
> We decided not to add anything related to the GW because, in the end, these are general functions,

I see, that makes sense. I was just worrying about informing users that these functions come from the GW and not the Core libs, but it's fine to leave them as they are (if you want, feel free to change the all uppercase to a just capitalized, but I'd say it's a minor thing, and we can probably go with what we have).